diff --git a/src/cof/cl_dll/cl_dll.h b/src/cof/cl_dll/cl_dll.h
index dcfb433..aa1aed5 100644
--- a/src/cof/cl_dll/cl_dll.h
+++ b/src/cof/cl_dll/cl_dll.h
@@ -29,6 +29,9 @@
 #if !defined(CL_DLL_H)
 #define CL_DLL_H
 #include "build.h"
+#ifdef __cplusplus
+#include <cstdint>
+#endif
 typedef unsigned char byte;
 typedef unsigned short word;
 typedef float vec_t;
diff --git a/src/cof/dlls/extdll.h b/src/cof/dlls/extdll.h
index 6b55f76..bae00b2 100644
--- a/src/cof/dlls/extdll.h
+++ b/src/cof/dlls/extdll.h
@@ -16,6 +16,9 @@
 #if !defined(EXTDLL_H)
 #define EXTDLL_H
 #include "build.h"
+#ifdef __cplusplus
+#include <cstdint>
+#endif
 //
 // Global header file for extension DLLs
 //
