diff --git a/src/cof/common/mathlib.h b/src/cof/common/mathlib.h
index 427d896..f68d6b0 100644
--- a/src/cof/common/mathlib.h
+++ b/src/cof/common/mathlib.h
@@ -17,6 +17,7 @@
 #if !defined(MATHLIB_H)
 #define MATHLIB_H
 #if !__cplusplus
+#include <stdint.h>
 #include <math.h>
 #if HAVE_TGMATH_H
 #include <tgmath.h>
@@ -25,6 +26,7 @@
 #if HAVE_CMATH
 #include <cmath>
 #else
+#include <cstdint>
 #include <math.h>
 #endif
 #endif // __cplusplus
diff --git a/src/cof/dlls/extdll.h b/src/cof/dlls/extdll.h
index 6b55f76..a4b4709 100644
--- a/src/cof/dlls/extdll.h
+++ b/src/cof/dlls/extdll.h
@@ -16,6 +16,7 @@
 #if !defined(EXTDLL_H)
 #define EXTDLL_H
 #include "build.h"
+#include <cstdint>
 //
 // Global header file for extension DLLs
 //
